src/config/ConfigDescriptions.hpp

```diff
@@ -1253,6 +1253,12 @@ inline static const std::vector<SConfigOptionDescription> CONFIG_OPTIONS = {
         .type        = CONFIG_OPTION_BOOL,
         .data        = SConfigOptionDescription::SBoolData{false},
     },
+    SConfigOptionDescription{
```

Comment:
is there any reason for this to be configurable? Shouldn't it be on by default? What are the caveats?

Reply:
I haven't noticed any regressions on my system. Though with the new behaviour, from X11 apps' point of view, the clipboard is changed twice for each single copying from Wayland: once when the actual copy is performed, and once when focus is switched to XWayland. I'm just worried this might break some apps or workflows that I'm unaware of.

If you think that's unlikely or acceptable, I'm ok with removing the config option or enablling it by default.

Reply:
unlikely. I'd just keep it hardcoded on

Reply:
@vaxerski done